packages/zwave-js/src/lib/commandclass/BarrierOperatorCC.ts

```diff
@@ -291,6 +372,15 @@ export class BarrierOperatorCCCapabilitiesReport extends BarrierOperatorCC {
 	public get supportedSignalTypes(): readonly SignalType[] {
 		return this._supportedSignalTypes;
 	}
+
+	public toLogEntry(): MessageOrCCLogEntry {
+		return {
+			...super.toLogEntry(),
+			message: {
+				"supported types": this.supportedSignalTypes,
```

Comment:
Needs to be enumerated too (and printed as a list). Look at MultilevelSensor, this has something similar.

Reply:
map and join... i think the next commit should be fixing that! thanks!